### PR TITLE
Update install-linux.md

### PR DESCRIPTION
**Pull request by draeath**
Address `--localstatedir` and `--prefix` build arguments as per https://github.com/singularityware/singularity/issues/1264

### CODE REVIEW

**Comment by vsoch** [comment]
This looks good to me - I am wondering if we should put this note also with the admin installation guide, as it is most likely the installer (user or admin) would want to know about this if an admin installing on a cluster --> https://github.com/singularityware/singularityware.github.io/blob/master/pages/docs/admin-docs/admin-guide.md#installation. What are your thoughts? The text itself looks good, and I'm grateful for the contribution so others know about this. @GodloveD do you want to give it a look over, and do you have thoughts about the user vs. admin docs location? I think we might want to write this full snippet in the admin section, and then reference (and link to it) from here, like "If you are installing Singularity on a shared resource with shared or multiple filesystems, you likely should specify `--localstatedir` and should follow instructions here (link to admin docs). That way, the average user doesn't get confused if it's not relevant, but the (non average or admin) can find the needed information in either place.

**Comment by GodloveD** [comment]
+1 adding this snippet to the admin section and linking it in the install section.

**Comment by vsoch** [comment]
cool thanks @GodloveD ! @draeath do those quick changes sound reasonable?

**Comment by draeath** [comment]
Yes, that's reasonable to me!

File: pages/docs/install/install-linux.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ note: The 'make install' is required to be run as root to get a properly install
 {% include asciicast.html source='install-singularity.js' uid='install-singularity' title='Install Singularity' author='vsochat@stanford.edu' %}
 
 
+### Prefix in special places
+
+If you build Singularity with a non-standard `--prefix` argument, please be sure to review the <a href="../admin-docs/admin-guide">admin guide</a> for details regarding the `--localstatedir` variable. This is especially important in environments utilizing shared filesystems.
```
Comment (vsoch) [inline pages/docs/install/install-linux.md]: The link here, to work on github pages, I think might need to be just `/admin-guide`, see here --> http://singularity.lbl.gov/admin-guide

**Comment by vsoch** [comment]
This looks great! After you are done with small tweaks let me know, and I can resolve the conflicts, and then we can merge.

**Comment by draeath** [comment]
Thanks, I fixed that up on my fork, so assuming CircleCI doesn't bark, that should be good.

I'm happy with what I've written, but of course any tweaks you may have I'd be happy to put in! I'm not 100% sure I got the URL path for the link from install-linux.md to admin-guide.md correct, as I'm not terribly familiar with the github.io platform. Definitely have a look at that before accepting.

EDIT: yes, I see, fixed the link!

**Comment by vsoch** [comment]
looks good, thanks for adding this! Merging!

**Pull request merged**